src/test/java/CheckTest.java

```diff
@@ -0,0 +1,60 @@
+import com.sciamus.contractanalyzer.application.CheckReportDTO;
+import com.sciamus.contractanalyzer.application.ContractChecksService;
+import com.sciamus.contractanalyzer.application.mapper.CheckReportMapper;
+import com.sciamus.contractanalyzer.domain.checks.rest.CheckRepository;
+import com.sciamus.contractanalyzer.domain.checks.rest.RestContractCheck;
+import com.sciamus.contractanalyzer.domain.checks.rest.dummy.DummyRestContractCheck;
+import com.sciamus.contractanalyzer.domain.checks.rest.reportcheck.CurrentUserService;
+import com.sciamus.contractanalyzer.domain.reporting.checks.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+import com.sciamus.contractanalyzer.domain.reporting.idGenerator.ReportIdGenerator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.keycloak.KeycloakSecurityContext;
+import org.keycloak.representations.AccessToken;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.net.MalformedURLException;
+import java.util.List;
+
+@ExtendWith(MockitoExtension.class)
+public class CheckTest {
+
+    @Mock
+    private KeycloakSecurityContext keycloakSecurityContextMock;
+
+    @Mock
+    private ReportRepository reportRepositoryMock;
+
+    private ContractChecksService contractChecksService;
+
+    @BeforeEach
+    public void init() {
+        final CurrentUserService currentUserService = new CurrentUserService(keycloakSecurityContextMock);
```

Comment:
## notice

Spróbujmy to zwinąć do deklaracji mocków i powołania obiektu jednym wywołaniem:
https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans-java-using-import

Czyli zależności z innych configów będą dostarczane poprzez wstrzykiwanie zależnych configów. W przypadku mocków, będzie conifuracja interfejsowa odpowiednio podmieniana mockami w testach

czyli potrzebujemy bean z zewnątrz, to do configa wstrzykujemy config adapterów (przez konstruktor!), a każda metoda wytwórcza będzie z tego configa korzystać :)